releases/UWP/README.md

```diff
@@ -60,12 +60,12 @@ When using Visual Studio this package requires Visual Studio 2015 Update 3 or la
     - System.Threading.Overlapped
     - System.Xml.XmlDocument
 
-### 5.2.2 (June 27, 2016)
+### 5.2.2 (July 14, 2016)
 When using Visual Studio this package requires Visual Studio 2015 Update 3 or later. You can read more about these changes in the Visual Studio 2015 Update 3 blog [post](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs).
```

Comment:
Is this true? I don't think 5.2.2 is coupled with Visual Studio 2015 Update 3. There is nothing prevents you from using it in previous toolchains (we don't block it).